rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostsFileBasedInfrastructureManager.java

```diff
@@ -558,7 +558,7 @@ protected String generateNodeName(HostTracker hostTracker) {
             String configuredAddress = hostTracker.getConfiguredAddress();
             String compliantAddress = convertToCompliantAddress(configuredAddress);
             return nodeSource.getName() + COMPLIANT_ADDRESS_DELIMITER + compliantAddress + COMPLIANT_ADDRESS_DELIMITER +
-                   ProActiveCounter.getUniqID();
+                   0;
```

Comment:
So, there will be one node name per host controlled by SSH ? I am worried about the case when compliantAdress will be shared among multiple host: In recently, I had the case of nodes using sporadically the IP address associated to docker0 NIC instead of eth0. The IP address associated to docker0 was the same on all VMs In this case, I had multiple node claiming the same IP while being located on different VMs. In similar condition, I am afraid your modification could lead to some collision in node name, and I am not sure how the platform could react. Can we consider using another identifier than compliantAdress ?